org.okstar.ok-msg.yml

```diff
@@ -0,0 +1,97 @@
+id: org.okstar.ok-msg
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: ok-msg-desktop
+rename-icon: ok-msg
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --device=all
+  - --talk-name=org.gnome.Mutter.IdleMonitor
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=com.canonical.indicator.application
+
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    add-ld-path: .
+    version: '23.08'
+    autodownload: true
+    autodelete: false
+
+build-options:
+  strip: false
+  no-debuginfo: true    
+  build-args:
+    - --share=network
+cleanup:
+  - /doc
+  - /include
+  - /lib/cmake
+  - /lib/libexec
+  - /lib/metatypes
+  - /lib/pkgconfig
+  - /lib/systemd
+  - /mkspecs
+  - /modules
+  - /share/doc
+  - /share/man
+  - '*.prl'
+  - '*.la'
+  - '*.a'
+
+modules:
+  - name: tcl
+    buildsystem: simple
+    build-commands:
+      - cp -r ../tcl /app
+      - cd /app/tcl/unix; ./configure --prefix=/app --exec-prefix=/app && make && make install
+
```

Comment:
this is not how you build tcl. 

1. /app is NEVER a builddir
2. you can just use `buidsystem: autotools` and `subdir: unix`

Reply:
1. Because tk dependence tcl，so at tk building stage that must to use tcl binaries.
2. I try to use autotools, but tk doesn't access to tcl binaries.
3. Would you show me doc that how to use `subdir: unix`.

Reply:
```
$ man flatpak-manifest
```

Reply:
I use  ` ${FLATPAK_DEST}` instead of /app  can resolve the problem, so this method is correct?

Reply:
this does change a single thing related to what I said to do. If you don't believe me, github has a search function that will show you how other package do it for better or worse. (hint: Inkscape does it right)